### PR TITLE
Add 11th user meeting to events

### PR DESCRIPTION
**Pull request by hflynn**
More porting of content from the old website

### CODE REVIEW

**Comment by hflynn** [comment]
Will be staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/

File: events/11th-annual-users-meeting-2016.html [hunk 1]
```diff
@@ -0,0 +1,307 @@
+---
+layout: news-events
+filename: events
+title: 11th Annual Users Meeting 2016
+main-blurb: The 11th Annual OME Users Meeting was held at the University of Dundee on May 31st to June 2nd 2016.
```
Comment (mtbc) [inline events/11th-annual-users-meeting-2016.html]: maybe "from" instead of "on" given the "to"?

**Comment by sbesson** [comment]
As this is new content, only request would be use HTTPS for all links to downloads.openmicroscopy.org. Once this is done, this looks good to me and should be ready to merge for the next website deployment.

**Comment by hflynn** [comment]
Merging for 5.4.0 release website update

**Pull request merged**